shells/shell.nix

```diff
@@ -33,7 +26,6 @@ mkShell {
     rustfmt
     openssl
     openssl.dev
-    mdbook
```

Comment:
Curious as to why the removal of mdbook? 
I guess it's not needed for most devs building apps, just us internal devs working on documentation.

Reply:
Yeh and it's pretty easy to do ` nix run nixpkgs#mdbook -- serve docs/getting-started/ --open` or similar